arduino/sensors/TMP102_TempatureSensor/TMP102_TempatureSensor.ino

```diff
@@ -0,0 +1,47 @@
+const int ALERT_PIN = A3;
```

Comment:
This should be under you includes, additionaly if you want to be more memory efficient you can use the preprocessor directive `#define` followed by your variable name `ALERT_PIN` then set your pin on it. The end result should be `#define ALERT_PIN A3`